python/lsst/summit/utils/tmaUtils.py

```diff
@@ -1002,6 +1002,41 @@ def _mergeData(self, data):
 
         return merged
 
+    def getEvent(self, dayObs, seqNum):
+        """Get a specific event for a given dayObs and seqNum.
+
+        Repeated calls for the same ``dayObs`` will use the cached data if the
+        day is in the past, and so will be much quicker. If the ``dayObs`` is
+        the current day then the EFD will be queried for new data for each
+        call, so a call which returns ```None` on the first try might return an
```

Comment:
```suggestion
        call, so a call which returns ``None`` on the first try might return an
```

Reply:
And actually, might even just be `None`.  I'm fuzzy on the different use cases for single vs double `.

Reply:
Okay, that was supposed to be a single-backquoted None.  Not sure how to invoke that with github markdown...

Reply:
I am equally fuzzy 😬 We can certainly both agree that three/one isn't the right format though 😛